logstash-input-syslog.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   # Gem dependencies
   s.add_runtime_dependency 'logstash', '>= 1.4.0', '< 2.0.0'
 
+  s.add_runtime_dependency 'atomic'
```

Comment:
Would it be better to use the `the ruby-concurrency` gem?
Since the atomic gem will or is merged with it?

Readme extract:
```
This library has been moved to https://github.com/ruby-concurrency/atomic.
It will be merged with the concurrent-ruby gem under the same org soon.
```